public/javascript/main.js

```diff
@@ -377,4 +377,16 @@ $(function(){
           $(this).data('chart', chart);
       }
   })
+
+  // Event tracking
+  // Clicks on elements with data-ga-track-click attribute will be recorded
+  function trackClick(eventDescription){
+    eventDescription = eventDescription || 'Undescribed event.';
+    ga('send', 'event', 'user interaction', eventDescription, document.title);
```

Comment:
Worth getting input from @zarino @wrightmartin @nick on what `eventCategory`, `eventAction`, `eventLabel`, and `eventValue` we should send here, with an eye to keeping it flexible enough for future types of event tracking, but also simple enough to draw useful conclusions via the Analytics reporting UI. [(Docs here)](https://developers.google.com/analytics/devguides/collection/analyticsjs/events)